examples/golang/actors/http-echo-tinygo/wit/deps.toml

```diff
@@ -0,0 +1,6 @@
+blobstore = "https://github.com/WebAssembly/wasi-blob-store/archive/main.tar.gz"
+logging = "https://github.com/WebAssembly/wasi-logging/archive/main.tar.gz"
+http = "https://github.com/wasmCloud/wasi-http/archive/main.tar.gz" # TODO: Switch to upstream once Wasm tooling compatible
+poll = "https://github.com/WebAssembly/wasi-poll/archive/main.tar.gz" # TODO: Remove once keyvalue is compatible
+# io = "https://github.com/WebAssembly/wasi-io/archive/main.tar.gz" # TODO: Remove once keyvalue is compatible
+io = "./overrides/io" # Pin to newer version of streams contract
```

Comment:
None of this will work with 0.80 (latest `main`), you should directly import from `crates/runtime/wit/deps` for now, but that would pull in resources, which I believe is not supported yet by the tooling (?)

So I guess there's nothing to do here (yet), just be aware that this will not actually run on the latest host?  @ricochet

Reply:
With the conversation from earlier today it looks like the move was to merge this for now and either use it with v79 or find out if it we can upgrade the example later? @connorsmith256 @ricochet are we aligned on this?

Reply:
Golang examples need to be pinned to v79. There isn't an easy way to do that with the current tooling. Unless we cut a `release-0.79` and reference this in the toml instead of `main`.

The alternative is to use relative paths since they allow for atomic commits with the host. To make this work for golang today (similar to the current version on this branch that was built with an older version of main) is to be in a half-way state where we keep the committed deps on an older version of the WIT definitions. Pinning to main is a weird place in a monorepo where it is the last commit on main.

My preference is the release branch approach as it's very clear about the state of the world.

Reply:
For this PR, then the idea here is to add a warning about the version. The average user is going to need to configure *wash* to use the right version `0.79.0`, with something like:

```console
WASMCLOUD_VERSION=v0.79.0 wash up
```

So that's what I'll put in the README for the example.

Reply:
PR has been updated -- I added a warning about the required version, and changed the `wash up` command